CHANGES.rst

```diff
@@ -19,8 +19,11 @@ New Features
   in accordance to the metadata entry. Lists without medium information
   are removed, until such information can be verified [#1626]
 
+- Added ability to set height of application widget using `show` method. [#1646]
```

Comment:
Belated comment: I don't think we render the change log into HTML, so this is fine for now but for future reference, use double backticks. Thanks!